lib/hooks/prerun/track.ts

```diff
@@ -16,8 +16,12 @@
  */
 import { Hook } from '@oclif/config';
 
+let trackResolve: (result: Promise<any>) => void;
+
 // note: trackPromise is subject to a Bluebird.timeout, defined in events.ts
-export let trackPromise: PromiseLike<void>;
+export const trackPromise = new Promise((resolve, _reject) => {
```

Comment:
There's no need to include unused params:
```suggestion
export const trackPromise = new Promise((resolve) => {
```